app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -840,6 +844,8 @@ class BrowserFragment :
             }
             sessionUseCases.reload(tab.id)
         }
+
+        TrackingProtection.hasEverChangedEtp.set(true)
```

Comment:
I have doubts here... I originally set a preference to follow this, but as it changes once for true it will always remain the same. Is this setting the value forever also for the metric, or should it be set along with the others when initializing Glean ?

Reply:
If this is only ever set once, you can make this metric a `lifetime: user` in the metrics.yaml and Glean will maintain the state for you and you shouldn't have to set it again. Be aware, that there currently isn't a way to "un-set" a user lifetime metric.

Reply:
Thanks! I added `lifetime: user` for this metric, we don't need to set it ever to `false`. 
One question, though... I did not see this in the documentation. Does it have a default value? Meaning if we do not set it, is it being sent with a default value?